packages/@n8n/ai-workflow-builder.ee/src/utils/test/stream-processor.test.ts

```diff
@@ -86,6 +86,25 @@ describe('stream-processor', () => {
 				expect(message.text).toBe('Last message to display');
 			});
 
+			it('should handle compact_messages with object content', () => {
+				const chunk = {
+					compact_messages: {
+						messages: [
+							{ content: 'First message' },
+							{ content: { type: 'text', data: 'some data' } }, // Object content
```

Comment:
The type here does not match the one in `processStreamChunk` (`data` instead of `text`): https://github.com/n8n-io/n8n/blob/6cd15a6a848bde27f3e334ce0f5210b9765611a7/packages/%40n8n/ai-workflow-builder.ee/src/utils/stream-processor.ts#L38

Reply:
thanks for catching. We need better typing here. Can try to extract the type here and reuse